packages/components/src/internal/components/project/ProjectNameSetting.tsx

```diff
@@ -71,33 +73,27 @@ export const ProjectNameSetting: FC<Props> = memo(props => {
                         </label>
                     </span>
                 </div>
-            </FormGroup>
+            </div>
 
-            <FormGroup controlId="project-name-prop-title">
-                <ControlLabel className="col-xs-12 col-sm-2 text-left">Project Label</ControlLabel>
+            <div className="form-group">
+                <label className="control-label col-xs-12 col-sm-2 text-left" htmlFor="project-label">
+                    Project Label
+                </label>
 
                 <div className="col-sm-10 col-md-5">
-                    {nameIsTitle ? (
-                        <FormControl
-                            autoComplete="off"
-                            disabled={nameIsTitle}
-                            key="controlled"
-                            name="title"
-                            type="text"
-                            value={nameIsTitle ? name : undefined}
-                        />
-                    ) : (
-                        <FormControl
-                            autoComplete="off"
-                            defaultValue={nameIsTitle ? name : defaultTitle}
-                            key="uncontrolled"
-                            name="title"
-                            onChange={_onTitleChange}
-                            type="text"
-                        />
-                    )}
+                    <input
```

Comment:
I was intentional in having this render as two different components; controlled v uncontrolled. The reason for the `key` here is that it forces React to remount the component rather than just assume it is the same component with different props. This now generates a console warning:

<img width="1716" alt="image" src="https://github.com/LabKey/labkey-ui-components/assets/3926239/f0f4bd59-c5dc-4243-b798-da752da1688c">

I recommend:

```tsx
<div className="col-sm-10 col-md-5">
    {nameIsTitle && (
        <input
            autoComplete="off"
            className="form-control"
            disabled
            key="controlled"
            name="title"
            type="text"
            value={name}
        />
    )}
    {!nameIsTitle && (
        <input
            autoComplete="off"
            className="form-control"
            defaultValue={defaultTitle}
            key="uncontrolled"
            name="title"
            onChange={_onTitleChange}
            type="text"
        />
    )}
</div>
```

Reply:
It might also work to use your implementation but pass `''` instead of `undefined` to value.

```tsx
value={nameIsTitle ? (name ?? '') : ''}
```